cmd/cubic-cli/cmd/transcribe.go

```diff
@@ -168,6 +162,49 @@ var transcribeCmd = &cobra.Command{
 			return fmt.Errorf("invalid option for outputFormat: '%v'", outputFormat)
 		}
 
+		// utterance-json must print to stdout
+		if outputFormat == "utterance-json" && resultsPath != "-" {
+			return fmt.Errorf("Aborting: --outputFormat utterance-json can only write to stdout")
+		}
+
+		// Check for overwritting an existing results file.
+		if resultsPath != "-" {
+			isFolder, isFile, exists, err := statsPath(resultsPath)
+			if err != nil {
+				return fmt.Errorf("Error while checkout --output destination: %v", err)
+			}
+
+			if listFile {
+				// --output needs to be a valid (existing) folder.
+				if !exists {
+					return fmt.Errorf("Aborting: --output must be an existing directory: %v", resultsPath)
+				}
+				if isFile {
+					return fmt.Errorf("Aborting: --output must be a directory: %v", resultsPath)
+				}
+				verbosePrintf(os.Stdout, "listFile seems to have a valid --output folder.\n")
+			} else {
+				// --output needs to be a non-existing file, with an existing parent folder.
+				if isFolder {
+					// Existing or not, it can't be a folder.
+					return fmt.Errorf("Aborting because --output '%s' is a directory, not a file", resultsPath)
+				} else if isFile {
+					if exists {
+						return fmt.Errorf("Aborting because --output '%s' already exists", resultsPath)
+					}
+					// Check parent dir exists
+					isFolder, _, exists, err := statsPath(filepath.Dir(resultsPath))
+					if err != nil {
+						return fmt.Errorf("Error while checkout --output destination: %v", err)
+					}
+					if !isFolder || !exists {
+						return fmt.Errorf("Aborting because folder '%s' doesn't exist", filepath.Dir(resultsPath))
+					}
+					verbosePrintf(os.Stdout, "!listFile, && isFile: seemes to have a valid --output file, with existing parent dir\n")
```

Comment:
I know it's verbosePrintf, but this seems like too much internal detail.  Maybe just say 
```suggestion
					verbosePrintf(os.Stdout, "Writing to --output file %s\n", resultsPath)
```

Reply:
I think a "Planning on writting to --output file" would be more appropriate, but that's fine.